src/client/components/Layout/Layout.js

```diff
@@ -22,28 +29,117 @@ class Layout extends React.Component
         super(props);
 
         this.state = {
-            language: navigator.language.slice(0,2) || 'en'
+            language: navigator.language.slice(0,2) || 'en',
+            menuItems: sidebarMenu,
+            user: [],
+            token: '',
+            isLoading: true,
+            activeMenu: 'overview',
         };
 
         this.i18nManager = new i18nManager(this.state.language);
     }
 
-    render() 
+    componentDidMount()
+    {
+        const obj = getFromStorage('botany-bay');
+
+        if (obj && obj.token)
+        {
+            const { token } = obj;
+    
+            fetch(`http://${window.location.host}/auth/api/token?token=` + token)
+                .then((res) => res.json())
+                .then((json) =>
+                {
+                    if (json.success)
+                    {
+                        this.setState({
+                            token,
+                            isLoading: false
+                        }, () =>
+                        {
+                            this.getUser();
+                        });
+                    } 
+                    else 
+                    {
+                        this.redirectToLogin();
+                    }
+                });
+        }
+        else 
+        {
+            this.redirectToLogin();
+        }
+    }
+
+    redirectToLogin()
+    {
+        window.location.replace(`http://${window.location.host}/auth/login`);
+    }
+
+    getUser()
+    {
+        fetch(`/user/api/token?token=${this.state.token}`).then((res) => res.json()).then((json) =>
+        {
+            console.log(json);
+
+            if (json.success)
+            {
+                //this.setState({ user: json.data.user, isLoading: false }, () =>
+                //{
+                    
+                    //this.getProjects();
+                //});
+            }
+        });
+    }
+
+    handleItemClick(name)
+    {
+        this.setState({ activeMenu: name });
+    }
+
+    render()
     {
+        const { isLoading, activeMenu, menuItems, user } = this.state;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['activeMenu' is assigned a value but never used.](https://app.codacy.com/gh/ShipyardSuite/dashboard/pullRequest?prid=5673418)